server/routes/post-hire-status.ts

```diff
@@ -10,57 +10,108 @@ import keycloak from '../keycloak';
 import logger from '../logger';
 import { getParticipantByID } from '../services/participants';
 import { getAssignCohort } from '../services/cohorts';
+import { postHireStatuses } from '../constants';
 
 const router = express.Router();
 
+type postHireStatusBody = {
+  body: {
+    participantIds: number[];
+    status: postHireStatuses;
+    data: {
+      graduationDate?: string;
+      unsuccessfulCohortDate?: string;
+    };
+  };
+  user: {
+    user_id: string;
```

Comment:
Add an ESLint directive here to prevent errors, potentially alongside a JSDoc comment at the top of the type to explain what this is used for (and by extension why it would need a snake case property name).

Reply:
Updated!